src/js/views/globals/search/patient-search_views.js

```diff
@@ -129,6 +129,9 @@ const ListView = CollectionView.extend({
   tagName: 'ul',
   serializeCollection: noop,
   childView: PicklistItem,
+  collectionEvents: {
+    'sync update reset': 'render',
```

Comment:
Without `sync` the picklist items render twice. Not sure if we need `reset`.

If I log these like:

```js
'sync': function() {console.log('sync');},
'update': function() {console.log('update');},
'reset': function() {console.log('reset');},
```

* `reset` is called when the search query length is less than 3 or the search input is cleared.
* `update` and then `sync` are called when a search api request is made. I can't tell what the difference is between those two.

Reply:
Maybe only the `sync` event is needed here? Are `update` and `reset` re-renders already handled automatically behind the scenes?